operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md

```diff
@@ -0,0 +1,112 @@
+# Change SMNP credentials on Leaf switches
+This procedure changes the SNMP credentials on management leaf switches in the system. Either a single leaf switch can be updated to use new SNMP credentials, or update all leaf switches in the system to use the same global SNMP credentials.
+
+**Note**: This procedure will not update the default SNMP credentials used when new leaf switches are added to the system. To update the default SNMP credentials for new hardware follow the [Update default Air-Cooled BMC and Leaf switch SNMP credentials](Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md) procedure.
```

Comment:
Add comma after hardware in second sentence

Reply:
Should be resolved fixed now.